tools/autograd/gen_autograd.py

```diff
@@ -191,21 +191,19 @@ def gen_autograd(aten_path, out, autograd_dir):
     template_path = os.path.join(autograd_dir, 'templates')
 
     # Generate VariableType.h/cpp
-    from .gen_variable_type import gen_variable_type
-    gen_variable_type(out, aten_decls, template_path)
+    if not disable_autograd:
+        from .gen_variable_type import gen_variable_type
+        gen_variable_type(out, aten_decls, template_path)
 
     # Generate Functions.h/cpp
     from .gen_autograd_functions import gen_autograd_functions_lib
     gen_autograd_functions_lib(
         out, autograd_functions, template_path)
 
-    # Load deprecated signatures
-    deprecated = load_deprecated_signatures(
-        aten_decls, os.path.join(autograd_dir, 'deprecated.yaml'))
```

Comment:
Oh? Is this dead? Joyous day!

Reply:
It's still used by gen_autograd_python code path :)
load_deprecated_signatures() doesn't seem to have side-effect so I assume it's safe to delete since the result is not used here.